daml-lf/spec/daml-lf-1.rst

```diff
@@ -1242,12 +1242,12 @@ Then we define *well-formed expressions*. ::
       Γ  ⊢  τ  :  ⋆      Γ  ⊢ₑ  τ
```

Comment:
`⊢ₑ` doesn't use a context. In addition, `τ` has to be a closed type (see "ToAny").

```suggestion
      ⊢  τ  :  ⋆      ⊢ₑ  τ
```

Reply:
Excellent catch. Merci.